src/bin/bat/clap_app.rs

```diff
@@ -422,11 +433,17 @@ pub fn build_app(interactive_output: bool) -> ClapApp<'static, 'static> {
                      '--style=\"..\"' option to the configuration file or export the \
                      BAT_STYLE environment variable (e.g.: export BAT_STYLE=\"..\").\n\n\
                      Possible values:\n\n  \
-                     * full: enables all available components (default).\n  \
-                     * auto: same as 'full', unless the output is piped.\n  \
+                     * auto: enables all components and 'header', unless the output is piped.
+                     * (default)\n  \
+                     * full: enables all available components.\n  \
                      * plain: disables all available components.\n  \
                      * changes: show Git modification markers.\n  \
-                     * header: show filenames before the content.\n  \
```

Comment:
I would prefer if we let `header` remain an alias for just `header-filename` rather than change its meaning.